pom.xml

```diff
@@ -63,6 +63,11 @@
     
     <dependencies>
         <dependency>
+        	<groupId>org.jenkins-ci.plugins.workflow</groupId>
+        	<artifactId>workflow-durable-task-step</artifactId>
+        	<version>2.10</version>
```

Comment:
This dependency actually pulls in multiple plugins. I understand the reason, but it was my concern about the approach without Pipeline patch